### PR TITLE
Fix incorrect package ID reference in README

### PR DESCRIPTION
**Pull request by anurse**

### CODE REVIEW

File: client-ts/signalr/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ JavaScript and TypeScript clients for SignalR for ASP.NET Core
 ## Installation
 
 ```bash
-npm install @aspnet/signalr-client
+npm install @aspnet/signalr
```
Comment (JamesNK) [inline client-ts/signalr/README.md]: I noticed line 39 refers to the package as `"@aspnet/signalR"`. Lower case the R?

**Comment by davidfowl** [comment]
Why is the build red?

**Comment by anurse** [comment]
Packages failing to resolve, maybe they got rolled off the necessary MyGet feeds?

**Comment by anurse** [comment]
Opening a new combine PR just to cover all the things approved for the brief preview1 update :). This was already approved anyway.

**Pull request closed without merging**